aws/client/default_retryer.go

```diff
@@ -45,7 +48,7 @@ func (d DefaultRetryer) RetryRules(r *request.Request) time.Duration {
 		retryCount = 8
 	}
 
-	delay := (1 << uint(retryCount)) * (rand.Intn(30) + minTime)
+	delay := (1 << uint(retryCount)) * (seededRand.Intn(minTime) + minTime)
```

Comment:
Thanks for reviewing this so promptly!
The current fixed value of `30` yields +100% random delay for non-throttling error retries (`minTime=30`) but just +3% of random delay for throttling error retries (`minTime=1000`). That's actually the crux of the problem.
If the API measures call rate over seconds (e.g. Route 53) then adding just 30, 60, 120, etc. milliseconds of random time to each retry delay won't do much in terms of lowering the global call rate as observed by the API.


translates to just 3% jitter with the current 1000 minTime for throttling errors. `1000` which is much too low